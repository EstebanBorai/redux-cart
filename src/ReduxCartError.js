class ReduxCartError extends Error {
	constructor(message) {
		super(message);
		this.name = 'redux-cart Error';
	}
}

export default ReduxCartError;
