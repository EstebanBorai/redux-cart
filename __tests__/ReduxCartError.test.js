import ReduxCartError from '../src/ReduxCartError';

describe.only('Redux Cart Error', () => {
  it('should return an error matching the snapshot', () => {
    expect(new ReduxCartError('Test Error')).toThrowErrorMatchingSnapshot();
  });
});
