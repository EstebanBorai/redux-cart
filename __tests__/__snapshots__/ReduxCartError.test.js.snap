// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Redux Cart Error should return an error matching the snapshot 1`] = `"received is not a function"`;
